### PR TITLE
Validation: descriptions 

### PR DESCRIPTION
**Pull request by SueADarby**
one file with a links issue included

### CODE REVIEW

File: dynamics-nav/Numeric-Functions.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Numeric Functions"
+description: Discusses Numeric functions including ABS, POWER, RANDOM, RANDOMIZE, and ROUND for Decimal, Integer, BigInteger.
```
Comment (MeeraDi) [inline dynamics-nav/Numeric-Functions.md:3]: for Decimal, Integer, and BigInteger.

File: dynamics-nav/Numeric-Property.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Numeric Property"
+description: Discusses the Numeric property. It sets a value that requires that users enter only numbers in the field.
```
Comment (MeeraDi) [inline dynamics-nav/Numeric-Property.md:3]: It sets a value that requires users to enter only numbers in the field.

File: dynamics-nav/OBJECTID-Function--Page-.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "OBJECTID Function (Page)"
+description: Describes the code and syntax for the OBJECTID unction and returns a string in the "Page xxx" format.
```
Comment (MeeraDi) [inline dynamics-nav/OBJECTID-Function--Page-.md:3]: OBJECTID function

File: dynamics-nav/Object-Level-Security.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Object-Level Security"
+description: Discusses Object-level security is the set of permissions on Dynamics NAV objects that constitute a permission set.
```
Comment (MeeraDi) [inline dynamics-nav/Object-Level-Security.md:3]: This is a confusing sentence.

File: dynamics-nav/Object-Specifications-and-Limitations.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Object Specifications and Limitations"
+description: Discusses object specifications and l limitations for the database, tables, pages and reports, and Codeunits.
```
Comment (MeeraDi) [inline dynamics-nav/Object-Specifications-and-Limitations.md:3]: Discusses object specifications and limitations for the database, tables, pages, reports, and Codeunits.

File: dynamics-nav/Occurrence-Property.md [hunk 6]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Occurrence Property"
+description: Discusses the occurrence property which sets a value that specifies whether an attribute must occur in the data that is being imported or exported.
```
Comment (MeeraDi) [inline dynamics-nav/Occurrence-Property.md:3]: Occurence property

File: dynamics-nav/OnAfterAssignField-Trigger.md [hunk 7]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "OnAfterAssignField Trigger"
+description: Discusses the OnAfterAssignField trigger which is executed after a field has been assigned a value and before it is validated and imported.
```
Comment (MeeraDi) [inline dynamics-nav/OnAfterAssignField-Trigger.md:3]: after a field is assigned

File: dynamics-nav/OnAfterInsertRecord-Trigger.md [hunk 8]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "OnAfterInsertRecord Trigger"
+description: Discusses the OnAfterInsertRecord trigger which is executed after a record has been inserted into a database table.
```
Comment (MeeraDi) [inline dynamics-nav/OnAfterInsertRecord-Trigger.md:3]: a record is inserted

File: dynamics-nav/OnAfterModifyRecord-Trigger.md [hunk 9]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "OnAfterModifyRecord Trigger"
+description: Discusses the OnAfterModifyRecord trigger which is executed after a record has been modified and applies to XMLports.
```
Comment (MeeraDi) [inline dynamics-nav/OnAfterModifyRecord-Trigger.md:3]: a record is modified

**Comment by SueADarby** [comment]
**Global effort to fix build validation errors**
@buck1ey 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Pull request merged**